src/main/java/org/immregistries/mismo/match/PatientMatchResult.java

```diff
@@ -0,0 +1,42 @@
+package org.immregistries.mismo.match;
+
+public class PatientMatchResult {
+
+  private PatientMatchDetermination determination = null;
+  private String signatureLevel0 = "";
```

Comment:
Maybe make the signature levels their own object, this would give us freedom to provide more than just a string in the future? Not sure if it's necessary, but to start with the object would just hold a String.